test-utils.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Why is this file in js and not ts?

Reply:
Because they're on the root directory and are just test utils? But I could make it a tsx, to be honest.

Reply:
We do have a util folder under src and test folder. I think it should go in either of those.

Reply:
Gotcha! Going to do this then 😄